pontoon/base/models.py

```diff
@@ -1372,7 +1386,10 @@ def available(self):
         """
         Available project locales belong to available projects.
         """
-        return self.filter(project__disabled=False, project__resources__isnull=False).distinct()
+        return self.filter(
+            project__disabled=False,
+            project__resources__isnull=False,
+        ).distinct()
```

Comment:
So I looked into this method and figured we only use it twice. In https://github.com/mozilla/pontoon/pull/1057/files#diff-c18ce63ddb8aca155c0553a0c1b174e3R832 and also in the `pontoon//teams/views.py` file. And, both of these method should exclude system projects.

So I suggest you change the `available()` method to `visible()` and call it in these two places.